app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseCategoryResource/Pages/ViewKnowledgeBaseCategory.php

```diff
@@ -56,6 +57,9 @@ public function infolist(Infolist $infolist): Infolist
                     ->schema([
                         TextEntry::make('name')
                             ->label('Name'),
+                        TextEntry::make('parentCategory.name')
+                            ->label('Parent Category')
+                            ->visible(fn (KnowledgeBaseCategory $record) => KnowledgeBaseSubcategory::active() && ! blank($record->parent_id)),
```

Comment:
There is no need to display this as sub-categories should only be visible in the list of the sub-categories of a parent category. Where the experience for editing can just be a modal.